server/index.js

```diff
@@ -20,18 +27,26 @@ initRoutes(app, db, config);
 if (process.env.NODE_ENV === 'development') {
   seed(app, db);
 }
+// Instance webpack and give the compilers (Configs)
+const compiler = webpack([clientConfig, serverConfig]);
 
-boot(app, db);
+compiler.apply(new FriendlyErrorsWebpackPlugin()); // Just to show better messages and erros
+
+app.use(webpackDevMiddleware(compiler, {
+  serverSideRender: true,
+}));
```

Comment:
A express middleware that allows to serve the content of webpack from memory and also delays request if webpack is still bundling.